test/storage-memory.cloudflare.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const assert = require('node:assert')
+const { after, before, describe, test } = require('node:test')
+const { unstable_dev } = require('wrangler') // eslint-disable-line camelcase
+const { promisify } = require('util')
+
+const sleep = promisify(setTimeout)
+
+describe('storage memory (cloudflare)', async () => {
+  let worker
+
+  before(async () => {
+    worker = await unstable_dev('test/cloudflare/helpers/worker.mjs')
+  })
+
+  after(async () => {
+    await worker.stop()
+  })
+
+  test('able to invalidate without error', async () => {
```

Comment:
This test doesn't assert that invalidation works, since the function in the helper returns always an empty object,  doesn't it?

What should or should not happen? Throwing/not throwing an error, returning a different value, other?

Reply:
Thanks, more clear now. Could you use `assert.doesNotThrow`?

Reply:
I used `doesNotReject` (since it's async) in a3741e5